lib/src/prices/price_kind.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../model/off_tagged.dart';
+
+/// Kind of the Price.
+enum PriceKind implements OffTagged {
+  @JsonValue('CONSUMPTION')
+  online(offTag: 'CONSUMPTION'),
```

Comment:
```suggestion
  consumption(offTag: 'CONSUMPTION'),
```
Let's use reasonable names, right?